EIPS/eip-2212.md

```diff
@@ -0,0 +1,271 @@
+---
+eip: 2212
+title: ERC-2212 Interest Earning Stakes
+author: Paul Razvan Berg (@PaulRBerg)
+discussions-to: https://github.com/ethereum/EIPs/issues/2212
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-07-26
+requires: 20
+---
+
+## Simple Summary
+
+We propose a new way to monetise decentralised apps that involves users staking tokens in exchange for a product or a service and creators earning interest on the pooled stakes through a continuous lending protocol such as [Compound](https://compound.finance).
+
+## Abstract
+
+The advent of decentralised lending brought to the web3 ecosystem a new primitive for designing business models. We herein describe a smart contract interface that accepts deposits ("stakes") in [ERC-20](https://eips.ethereum.org/EIPS/eip-20) tokens. These deposits are immediately lent on the market to earn interest either for just the owner of the contract, or for both them and the user. We assume that users get access to a product or a service for as long as they keep staking tokens.
+
+## Motivation
+
+While most web3 business models rely on paying a percentage fee on transfers, or committing to a monthly subscription, we aim to flip the model on its head. Users are their own banks now, so, instead of charging them, we make them stake tokens that can be claimed back, in full, at any point in time. We view this as a win-win-win scenario for users, dapp creators and lending protocols.
+
+It is worth it to mention PoolTogether, a "no-loss lottery" that launched recently. It works by having participants deposit money in their contracts, locking it up for a while and earning interest through Compound. Ultimately, they choose a lucky winner and return the deposits back to all other players. PoolTogether takes a 10% commission on the prize.
+
+We wrote this specification because interest earning stakes are a powerful financial instrument and a community-vetted standard and implementation will reduce the cost of replication.
+
+### Use Cases
+
+- Software products and services
+- Charity pools
+- Games
+- Gym memberships
+
+## Specification
+
+Assigning an owner is required for this spec to make sense, thus we assume that a proper implementation would use a
+well-known contract such as OpenZeppelin's
+[Ownable](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/ownership/Ownable.sol). For
+brevity, we haven't included any method for assigning or transferring ownership.
+
+Every time we refer to cTokens, we mean Compound Tokens, as defined in the official
+[documentation](https://compound.finance/ctokens). These are ERC-20 tokens with an interest accrual feature built in.
```

Comment:
This link is now dead.

Reply:
Is it? I just opened it and it worked for me.

Reply:
It doesn't go to any kind of documentation, just a marketing page. There should be a link to a specification, if it cannot be another ERC.

Reply:
Fair enough.